File: ios/RNMBX/RNMBXTerrainManager.m
#import <React/RCTBridgeModule.h>
#import <React/RCTViewManager.h>

@interface RNMBXTerrain : UIView
{}
@property (nonatomic) NSObject* exaggeration;
@end

@interface RCT_EXTERN_MODULE(RNMBXTerrainManager, RCTViewManager)

RCT_EXPORT_VIEW_PROPERTY(sourceID, NSString);

RCT_EXPORT_VIEW_PROPERTY(reactStyle, NSDictionary);

@end

File: ios/RNMBX/RNMBXTerrainManager.swift
@objc(RNMBXTerrainManager)
class RNMBXTerrainManager: RCTViewManager {
    @objc
    override static func requiresMainQueueSetup() -> Bool {
        return true
    }
  
    override func view() -> UIView! {
      let terrain = RNMBXTerrain()
      terrain.bridge = self.bridge
      return terrain
    }
}
